blueocean-dashboard/src/main/js/creation/github/GithubFlowManager.js

```diff
@@ -140,7 +117,7 @@ export default class GithubFlowManager extends FlowManager {
 
     getApiUrl() {
         // backend will default to api.github.com
-        return null;
+        return 'https://api.github.com';
```

Comment:
Any reason to change it from null? I've noticed that when `apiUrl=https://api.github.com` is sent to creation endpoint, it adds it as an endpoint but this always seems odd to me.